topics/ui/controls/combo_box.md

```diff
@@ -23,79 +23,86 @@ comboBox.setEditable(true);
 This way, users won't waste their time clicking the empty combo box to find out that there are no options available.
-
-| <format color="Red" style="bold">Incorrect</format> | <format color="Green" style="bold">Correct</format> |
-|-----------------------------------------------------|-----------------------------------------------------|
-| ![](combo_box_empty.png){width="221"}               | ![](replace_with_iput_field.png){width="216"}       |
-
-Replace the input field with a combo box after a value has been entered and confirmed.
-
-### Menu
-
-#### Control
-
-Open the combo box menu by clicking the arrow button on the right or pressing the <shortcut>Down</shortcut> arrow key when the combo box is focused.
+  
+  <table style="none" border="false">
+    <tr>
+      <td width="50%">
+        <format color="Green" style="bold">Correct</format><img src="combobox_how_to_use_1_correct.png" alt="A correct example of a repository input field without values"/>
+      </td>
+      <td width="50%">
+        <format color="Red" style="bold">Incorrect</format><img src="combobox_how_to_use_1_incorrect.png" alt="An incorrect example of a repository input field without values"/>
+      </td>
+    </tr>
+  </table>
+
+* Replace the input field with a combo-box after a value has been entered and confirmed.
+
+### Behaviour
+
+1. Open the combo-box menu by clicking the arrow button on the right or pressing the <shortcut>Down</shortcut> arrow key when the combo box is focused.
 The menu opens down by default. If there is not enough space, the menu opens up.
 
-When the menu opens, select the element that was shown in the closed combo box. If a custom value is entered, then do not select a value in the list.
+2. When the menu opens, select the element that was shown in the closed combo-box. If a custom value is entered, then do not select a value in the list.
 
-Move the selection in the menu and update the value in the combo box by pressing the <control>Up</control> and <control>Down</control> arrow keys.
+3. Move the selection in the menu and update the value in the combo-box by pressing the <control>Up</control> and <control>Down</control> arrow keys.
 On mouse hover, move the selection to an item the cursor is pointing to and update the value by clicking the mouse button or pressing <shortcut>Enter</shortcut>.
 
-The menu remains opened until the user clicks the item in the list, presses <shortcut>Enter</shortcut> or <shortcut>Esc</shortcut>, clicks outside the menu,
+4. The menu remains opened until the user clicks the item in the list, presses <shortcut>Enter</shortcut> or <shortcut>Esc</shortcut>, clicks outside the menu,
 or switches to another app.
 
-#### Menu items
+### Menu items
 
-The menu list contains predefined or the most likely options. Follow the rules for [drop-down menu items](drop_down.md#menu-items).
+* The menu list contains predefined or the most likely options. Follow the rules for [drop-down menu items](drop_down.md#menu-items).
```

Comment:
Please rewrite as an instruction, e.g. "Show the predefined or most likely options in the list"